src/RunProcessAsTask.Tests/ProcessExTests.cs

```diff
@@ -60,13 +60,16 @@ public void WhenProcessRunsNormally_ReturnsExpectedResults()
                 Assert.True(results.RunTime.TotalMilliseconds >= millisecondsToSleep);
             }
 
-            [Fact]
-            public void RunLotsOfOutputForFiveMinutes()
+            [Theory]
+            [InlineData(10)]
+            //[InlineData(60)]
+            //[InlineData(300)]
+            public void RunLotsOfOutputForPeriod(int seconds)
             {
                 // when this problem manifested with the older code, it would normally 
                 // trigger in this test within 5 to 10 seconds, so if it can run for 
                 // 5 minutes and not cause the output-truncation issue, we are probably fine
-                var fiveMinutes = TimeSpan.FromMinutes(5);
+                var fiveMinutes = TimeSpan.FromSeconds(seconds);

```

Comment:
Given the comment, 10s is definitely too little.

Reply:
@EamonNerbonne This is for CI, 5 min is too long for a single CI test

Reply:
https://docs.travis-ci.com/user/customizing-the-build#Build-Timeouts
https://www.appveyor.com/docs/build-configuration/#build-timeout

5min does not appear to be close to the limits; but there's a large gap below 5min above 10s - and if the bug is documented to *usually* occur between 5s and 10s, then stopping after 10s is asking for a flaky test.

note that this is in some sense a moot point; because the code is currently OK, and it's rather unlikely it'll accidentally be reintroduced.